cpp/include/raft/linalg/strided_reduction.cuh

```diff
@@ -71,6 +74,84 @@ void stridedReduction(OutType* dots,
   detail::stridedReduction(dots, data, D, N, init, stream, inplace, main_op, reduce_op, final_op);
 }
 
+/**
+ * @defgroup strided_reduction Strided Memory Access Reductions
```

Comment:
The capitalization here seems a little weird. Is this a title?

Reply:
Yes it's the title for the group

Reply:
Yes, it's the title of the doc group